spacy/lang/grc/lex_attrs.py

```diff
@@ -0,0 +1,309 @@
+from ...attrs import LIKE_NUM
+
+
+_num_words = [
+    # CARDINALS
+    "εἷς",
+    "ἑνός",
+    "ἑνί",
+    "ἕνα",
+    "μία",
+    "μιᾶς",
+    "μιᾷ",
+    "μίαν",
+    "ἕν",
+    "δύο",
+    "δυοῖν",
+    "τρεῖς",
+    "τριῶν",
+    "τρισί",
+    "τρία",
+    "τέτταρες",
+    "τεττάρων",
+    "τέτταρσι",
+    "τέτταρα",
+    "τέτταρας" "πέντε",
```

Comment:
Here reformatting with `black` shows where the commas are missing. (This is a very common typo in these kinds of lists, no worries!)

Reply:
I ran black on it, bu I did not see any.

Reply:
Now I see how black works. Thanks.